### PR TITLE
Closes #159 adlb_neuro metadata [create website]

### LINKED ISSUE #159: Feature Request:  Add adlb_neuro dataset to pharmaverseadam
### Feature Idea

Add adlb_neuro dataset to pharmaverseadam

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by MilesAlmond**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/cran-release/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [X] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [X] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [X] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [X] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [X] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [X] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

To create the website for the content of this PR add "[create website]" to the title of the pull request.

### CODE REVIEW

**Comment by MilesAlmond** [comment]
Will need to wait for ADLB_NEURO to be added to package before merging (preview-adam vignette will not run without it)

**Comment by MilesAlmond** [comment]
@Fanny-Gautier - Hi Fanny, I think I am done except that I also need to update adams-specs.json. Is there a function I should be using to create this from adams-specs.xlsx?

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier - Hi Fanny, I think I am done except that I also need to update adams-specs.json. Is there a function I should be using to create this from adams-specs.xlsx?

Hi @MilesAlmond, thank you for the updates. To create the `.json` file you will need to run the beginning of the program `pharmaverseadam/data-raw/create_adams_data.R`. Please let me know if you face any issue. Thank you.

**Comment by MilesAlmond** [comment]
Ready for review

**Comment by Fanny-Gautier** [comment]
Metadata looks ok, thank you @MilesAlmond for the implementation.
@meilinjiang Could you please confirm when is the `{admiralneuro}` release expected, so that we can fully integrate `ADLB_neuro` to `{pharmaverseadam}`? Thank you

**Comment by meilinjiang** [comment]
> Metadata looks ok, thank you @MilesAlmond for the implementation. @meilinjiang Could you please confirm when is the `{admiralneuro}` release expected, so that we can fully integrate `ADLB_neuro` to `{pharmaverseadam}`? Thank you

Thank you Miles and Fanny! CRAN package submissions are officially closed and on hold through August 19 for their team's summer vacation and system maintenance. Will attempt to submit again this week.

File: inst/extdata/adams-specs.json [hunk 1]
```diff
@@ -345,6 +345,17 @@
       "Has No Data": "No",
       "Repeating": "Yes",
       "Reference Data": "No"
+    },
+    {
+      "Dataset": "ADLB_N",
+      "Label": "Laboratory Analysis for Neuroscience",
+      "Class": "BASIC DATA STRUCTURE",
+      "Structure": "One or more records per subject per analysis parameter per analysis timepoint",
+      "Key Variables": "STUDYID, USUBJID, PARAM, PARAMCD, AVAL",
```
Comment (Fanny-Gautier) [inline inst/extdata/adams-specs.json:354]: will be updated as necessary in #168. I believe that AVAL is not correct for Analysis Timepoint.

Comment (manciniedoardo) [reply inst/extdata/adams-specs.json:354]: yeah usually AVAL is not a key. Do we not have AVISIT/ATPT?

**Review by manciniedoardo — CHANGES_REQUESTED** [review]

**Comment by Fanny-Gautier** [comment]
@Lina2689 Could you please review once you get a chance? Many thanks
